File: bootstrap/src/main/java/net/dontdrinkandroot/wicket/bootstrap/component/item/ItemContainer.java
package net.dontdrinkandroot.wicket.bootstrap.component.item;

import net.dontdrinkandroot.wicket.css.CssClass;

public interface ItemContainer
{
    default CssClass getItemClass()
    {
        return null;
    }

    default CssClass getLinkClass()
    {
        return null;
    }
}

File: bootstrap/src/main/kotlin/net/dontdrinkandroot/wicket/bootstrap/component/item/ItemContainer.kt
package net.dontdrinkandroot.wicket.bootstrap.component.item

import net.dontdrinkandroot.wicket.css.CssClass
import org.apache.wicket.markup.repeater.RepeatingView

interface ItemContainer {

    val itemClass: CssClass?
        get() = null

    val linkClass: CssClass?
        get() = null
}

class ItemView(id: String) : RepeatingView(id)